packages/nimble-components/src/wafer-map/template.ts

```diff
@@ -24,5 +24,8 @@ export const template = html<WaferMap>`
         <div class="wafer-map-area">
             <canvas class="wafer-map-canvas" ${ref('canvas')}></canvas>
         </div>
+        <svg class="hover-layer">
```

Comment:
Does this new layer (and associated CSS) need to be part of this PR or can it wait for the hover PR (possibly with a TODO instead if that is helpful for tracking)?

Reply:
Yup, I think it can be removed until the next PR.